src/util.cr

```diff
@@ -60,8 +60,8 @@ module Rlp::Util
   # ```
   def self.bin_to_hex(b : Bytes)
     h = b.hexstring
-    h = "0#{h}" if h.size.odd?
-    return h
+    return "0#{h}" if h.size.odd?
+    h
```

Comment:
this doesn't look correct. bug in the linter?

Reply:
That's on purpose. The linter actually only suggested to remove `return`.

I used `return` to break out early without changing `h`. https://devblast.com/b/what-are-guard-clauses